### PR TITLE
Add new node with k6 for load tests, fixes #271

### LINKED ISSUE #271: Add a load node
The "test_custom_load" test requires a load node to work.

The dev-env needs a target in make that will up one container and add an record to the host file.
This container will be the load node in the test cluster.

### PR DESCRIPTION
**Pull request by evgeniiz321**

### CODE REVIEW

File: k6_node/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM lscr.io/linuxserver/openssh-server:latest
```
[the remark this answers is not in the document]
Comment (evgeniiz321) [reply k6_node/Dockerfile:1]: Current load tests expect to have an ssh connection to the load node. There is too much code to change in tests if connect somehow different.

File: Makefile [hunk 2]
```diff
@@ -23,9 +23,12 @@ include services/*/prepare.mk
 START_SVCS = $(shell cat .services | grep -v '\#')
 START_BASIC = $(shell cat .basic_services | grep -ve '\#')
 START_BOOTSTRAP = $(shell cat .bootstrap_services | grep -v '\#')
+START_TESTING = $(shell cat .testing_services | grep -v '\#')
 STOP_SVCS = $(shell tac .services | grep -v '\#')
 STOP_BASIC = $(shell tac .basic_services | grep -v '\#')
 STOP_BOOTSTRAP = $(shell tac .bootstrap_services | grep -v '\#')
+STOP_TESTING = $(shell tac .testing_services | grep -v '\#')
+
```
[the remark this answers is not in the document]
Comment (evgeniiz321) [reply Makefile:31]: removed

**Review by roman-khimov — APPROVED** [review]

File: k6_node/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    hostname: k6-node
+    container_name: k6-node
+    restart: on-failure
+    networks:
+      k6_node_int:
+      internet:
+        ipv4_address: ${IPV4_PREFIX}.102
+    stop_signal: SIGKILL
+    env_file: [ ".env" ]
+    volumes:
```
Comment (vvarg229) [inline k6_node/docker-compose.yml:21]: I think it's a good idea to add here hosts file:
`- ./../../vendor/hosts:/etc/hosts`

**Review by vvarg229 — CHANGES_REQUESTED** [review]
Did you run a load test locally with this dev-env?

File: Makefile [hunk 4]
```diff
@@ -241,4 +261,7 @@ prepare-test-env:
 	echo "Step 4: Preparing the storage service..."; \
 	$(MAKE) prepare.storage; \
 
+	echo "Step 5: Preparing the k6 load node..."; \
+	$(MAKE) up/testing; \
```
Comment (vvarg229) [inline Makefile:265]: Should we wait for some time or check something after starting a k6 container?

File: k6_node/artifacts.mk [hunk 5]
```diff
@@ -0,0 +1,5 @@
+K6_NODE_DIR=$(abspath services/k6_node)
+
+get.k6_node:
+	@echo "⇒ Creating keys for k6 node server and clients"
+	${K6_NODE_DIR}/generate_keys.sh > /dev/null
```
Comment (vvarg229) [inline k6_node/artifacts.mk:5]: And how will work with tests in the next step?
Here you have generated keys... What's next?

File: k6_node/.env
[the remark this answers is not in the document]
Comment (evgeniiz321) [reply k6_node/.env:1]: Yes, didn't notice. Added symlink + made a separate env file for service specific variables

File: k6_node/generate_keys.sh [hunk 6]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (evgeniiz321) [reply k6_node/generate_keys.sh]: Done

File: k6_node/docker-compose.yml (hunk 3 above)
Comment (evgeniiz321) [reply k6_node/docker-compose.yml]: Added

File: Makefile (hunk 4 above)
Comment (evgeniiz321) [reply Makefile:265]: There is nothing to wait there, but I added 5 secs just in case

File: k6_node/artifacts.mk (hunk 5 above)
Comment (evgeniiz321) [reply k6_node/artifacts.mk]: Path to private key will be specified in variables for load test

**Comment by evgeniiz321** [comment]
Yes, I tested it with an updated version of load test:

`(venv.local-pytest) testsrv1% pytest pytest_tests/testsuites/load/test_load.py
================================================================================================ test session starts =================================================================================================
platform linux -- Python 3.10.0, pytest-7.3.1, pluggy-1.0.0
rootdir: /home/ezayats/neofs-testcases/pytest_tests
configfile: pytest.ini
plugins: allure-pytest-2.9.45, lazy-fixture-0.6.3
collected 1 item

pytest_tests/testsuites/load/test_load.py::TestLoad::test_custom_load[1-4-grpc-1-4-200-200-0-0-1000-1mb_200.json] PASSED                                                                                       [100%]`

**Review by roman-khimov — APPROVED** [review]

**Review by vvarg229 — APPROVED** [review]

**Pull request merged**